src/components/accessories/infoBox/types.ts

```diff
@@ -1,4 +1,10 @@
 export interface IProps {
-  type: "warning" | "error";
+  type: "info" | "warning" | "error";
   message: string;
 }
+
+export enum InfoBoxType {
+  Info = "info",
+  Warning = "warning",
+  Error = "error",
+}
```

Comment:
I think that should be more clean if we remove the static string on the IProps interface and use instead the belowed enum.

Reply:
Ok. I have applied your changes. In order to avoid changing all the components to use the enum, I set the type to be enum or string, you can recheck.